modules/AlphaWalletCore/AlphaWalletCore/WebImageURL.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  AlphaWalletURL.swift
+//  AlphaWalletCore
+//
+//  Created by Vladyslav Shepitko on 11.02.2022.
+//
+
+import Foundation
+
+public enum WebImageURL: Codable, Hashable, Equatable, CustomStringConvertible {
+    case origin(URL)
+    case googleContentRewritten(URL)
```

Comment:
Are we rewriting the URL for Google hosted images for all cases? Should we add a case or somehow be able to choose when to use the original. Eg. we only want to use the rewritten, bigger version when in Wallet > Collectibles, and Info, and in the screen showing a single token ID, but not in screens like the little icons in Wallet tab or Activity Tab. 

Concerned with memory usage and speed.